code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+
+/obj/item/slimecross/destabilized/cerulean
+	colour = "cerulean"
+
+/obj/item/slimecross/destabilized/cerulean/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(ismovableatom(A))
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), A, thrower)
+	else
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), thrower, thrower)
+
+/obj/item/slimecross/destabilized/pyrite
+	colour = "pyrite"
+
+/obj/item/slimecross/destabilized/pyrite/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/turf/T = get_turf(A)
+	T.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+	for(var/atom/atoms in T)
+		atoms.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+
+/obj/item/slimecross/destabilized/red
+	colour = "red"
+
+/obj/item/slimecross/destabilized/red/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.apply_status_effect(/datum/status_effect/heavyBleeding)
+	else
+		new /obj/effect/decal/cleanable/blood(get_turf(A))
+
+/obj/item/slimecross/destabilized/green
+	colour = "green"
+
+/obj/item/slimecross/destabilized/green/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A) && isliving(thrower))
+		var/mob/living/LTarget = A
+		var/mob/living/LThrower = thrower
+		var/damageToDeal
+		if(iscarbon(LThrower))
+			var/mob/living/carbon/CThrower = LThrower
+			var/damageType = findBiggestLoss(CThrower)
+			damageToDeal = min(CThrower.get_damage_amount(damageType), 25)
+			CThrower.apply_damage_type(-25, damageType)
+			if(iscarbon(LTarget))
+				var/mob/living/carbon/CTarget = LTarget
+				CTarget.apply_damage_type(damageToDeal, damageType)
+				return
+			LTarget.adjustBruteLoss(damageToDeal)
+			return
+		damageToDeal = -(LThrower.adjustBruteLoss(-25))
+		LTarget.adjustBruteLoss(damageToDeal)
+
+/obj/item/slimecross/destabilized/green/proc/findBiggestLoss(mob/living/carbon/C)
+	var/damageType = BRUTE
+	var/maxDamage = C.get_damage_amount(BRUTE)
+	var/burnDamage = C.get_damage_amount(BURN)
+	if(burnDamage > maxDamage)
+		maxDamage = burnDamage
+		damageType = BURN
+	var/toxinDamage = C.get_damage_amount(TOX)
+	if(toxinDamage > maxDamage)
+		maxDamage = toxinDamage
+		damageType = TOX
+	var/oxyDamage = C.get_damage_amount(OXY)
+	if(oxyDamage > maxDamage)
+		maxDamage = oxyDamage
+		damageType = OXY
+	var/cloneDamage = C.get_damage_amount(CLONE)
+	if(cloneDamage > maxDamage)
+		maxDamage = cloneDamage
+		damageType = CLONE
+	var/staminaDamage = C.get_damage_amount(STAMINA)
+	if(staminaDamage > maxDamage)
+		maxDamage = staminaDamage
+		damageType = STAMINA
+	var/brainDamage = C.get_damage_amount(BRAIN)
+	if(brainDamage > maxDamage)
+		maxDamage = brainDamage
+		damageType = BRAIN
+	return damageType
+
+/obj/item/slimecross/destabilized/pink
+	colour = "pink"
+
+/obj/item/slimecross/destabilized/pink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(!isliving(A) || !iscarbon(thrower))
+		return FALSE //Don't use up the extract unless it lands a hit on a eligible mob.
```

Comment:
while cute, I think this crossbreed extract's effect is very weak. maybe make it infinitely reusable, regardless of whether or not it hits something?

Reply:
Fair enough